consensus/consensus-types/src/block_retrieval.rs

```diff
@@ -8,6 +8,8 @@ use libra_types::validator_verifier::ValidatorVerifier;
 use serde::{Deserialize, Serialize};
 use std::fmt;
 
+pub const MAX_BLOCKS_PER_REQUEST: u64 = 10;
```

Comment:
do we want to add this to a config? do we have any metrics on average / max blocks per request on any network?

Reply:
Also curious how to come up with this number?  400 bytes * 1000 * 10 = 4MB?  Seems reasonable.

Reply:
random small number :)